.pre-commit-hooks.yaml

```
@@ -15,3 +15,12 @@
   language: golang
   name: Helm Docs Built
   require_serial: true
+
+- id: helm-docs-container
+  args: []
+  description: Uses the container image of 'helm-docs' to create documentation from the Helm chart's 'values.yaml' file, and inserts the result into a corresponding 'README.md' file.
+  entry: jnorwood/helm-docs:v1.11.0
```

As the problems that were happening with v1.11.1 were fixed, I think you should just define it as latest.
```suggestion
  entry: jnorwood/helm-docs:latest

---

Based on feedback from the `pre-commit` developer, I think the intent it to pin the version to be at the same release as the respective git commit. Meaning that when you use the `golang` version, for example, you are always using the code pinned to the git tag. So `v1.11.0` would be "immutable" with respect to git vs docker.

That being said, it would mean updating the `.pre-commit-hooks.yaml` every time a new docker image is released. I'm personally on the fence about whether this is good or bad for various reasons.

Leaving it at `latest` makes a lot of sense to me from a bug fix perspective if the user doesn't prefer a specific version, but breaks with the pattern of intent around versioning from `pre-commit` itself. There is also the overhead in bumping the tag regularly or it getting out of sync, etc.

I am perfectly willing to accept that the docker image is always `latest` unless you change it to something specific in the respective git repos.

Since this is your project, I'll go with your opinion. I just wanted to extend the conversation about it a bit before committing one way or the other.

Let me know what direction you want to go in, and I'll update the README to clarify usage as needed.

---

Hey. Sorry about the late response. Got some time today and saw your latest ping. I really liked your approach that defines it to on default use the latest tag but give the user capability to pin it.